package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@theia/che-theia-hosted-plugin-manager-extension",
+  "keywords": [
+    "theia-extension"
+  ],
+  "version": "0.3.8",
+  "files": [
+    "lib",
+    "node"
+  ],
+  "dependencies": {
+    "@theia/core": "latest",
+    "@theia/plugin-ext": "latest",
+    "@eclipse-che/workspace-client": "latest"
+  },
+  "devDependencies": {
+    "rimraf": "^2.6.2",
+    "typescript": "latest"
```

Comment:
Set up please 2.7.2 version. For this one version CQ passed.

Reply:
ok